app/controllers/devise_token_auth/omniauth_callbacks_controller.rb

```diff
@@ -13,7 +13,7 @@ def redirect_callbacks
       # before authentication.
       devise_mapping = [request.env['omniauth.params']['namespace_name'],
                         request.env['omniauth.params']['resource_class'].underscore.gsub('/', '_')].compact.join('_')
-      redirect_route = "#{request.protocol}#{request.host_with_port}/#{Devise.mappings[devise_mapping.to_sym].fullpath}/#{params[:provider]}/callback"
+      redirect_route = "#{request.protocol}#{request.host_with_port}/#{Rails.application.config.relative_url_root}/#{Devise.mappings[devise_mapping.to_sym].fullpath}/#{params[:provider]}/callback"
```

Comment:
@okuramasafumi - shouldn't we be doing the same `relative_url_root.present?` check here, as well, so as not to generate double-slashes?